mojap_metadata/metadata/specs/table_schema.json

```diff
@@ -377,6 +377,17 @@
             "type": "string",
             "title": "Path to your data can be a folder path, filepath or URL."
         },
+        "glue_table_properties": {
+            "type": "object",
+            "propertyNames": {
+                "pattern": "^[\u0020-\uD7FF\uE000-\uFFFD\t]*$",
+                "maxLength": 255
+            },
+            "additionalProperties": {
+                "type": "string",
+                "maxLength": 512000
+            }
+        },
```

Comment:
```suggestion
        "glue_table_properties_custom": {
            "type": "object",
            "propertyNames": {
                "pattern": "^[\u0020-\uD7FF\uE000-\uFFFD\t]*$",
                "maxLength": 255
            },
            "additionalProperties": {
                "not": {
                    "type": "string",
                    "pattern": ".*[,].*"
                },
                "maxLength": 512000
            }
        },
        "glue_table_properties_aws": {
            "type": "object"
```

Reply:
Please see below comment to update the v1.5.0 schema in metadata_schema which will resolve the failing test.